docker-compose.sample.yml

```diff
@@ -68,8 +69,22 @@ services:
     #
     # Full list : https://github.com/hardware/mailserver#environment-variables
     #
+    labels:
+      - "traefik.enable=true"
+      - "traefik.docker.network=http_network"
+      - "traefik.http.routers.spam.entrypoints=websecure"
+      - "traefik.http.routers.spam.rule=Host(`spam.${MAILSERVER_DOMAIN}`)"
+      - "traefik.http.routers.spam.service=spam"
+      - "traefik.http.routers.spam.tls=true"
+      - "traefik.http.routers.spam.tls.certresolver=letsencrypt"
+      - "traefik.http.routers.spam.tls.domains[0].main=${MAILSERVER_HOSTNAME}.${MAILSERVER_DOMAIN}"
```

Comment:
Q: if you have multiple domains, you need to add here or somewhere else? (additional domains)

Reply:
Could you please elaborate? Does rspamd spawn an instance per domain and/or accessible on different URLs when more than one domain?

Reply:
Sure, In Traefik 1.x I used this option: https://github.com/mailserver2/mailserver/blob/master/docker-compose.sample.yml#L60

Can we still use this option in Traefik 2.x?

Reply:
Sorry, I still cannot see the connection. How does `ADD_DOMAINS` relate to this configuration you are commenting about?

Reply:
Or, if I try to re-phrase that, what do you think will be broken, if the proposed change is not made (if anything)?

Reply:
I am just thinking about the case when you have one mail server on your host and you have multiple domains (virtual domains) on your mail server.  Do you need to change anything in the traefik setup in the V2?  This is just an upgrade question from v1 to v2.  (For myself, I have multiple additional domains set up next to my main domain)

Reply:
> Do you need to change anything in the traefik setup in the V2?

Do you have reasons to believe that you do? If so, what are those reasons?

Reply:
So I just slept on it, and I think I might know what you are trying to get at. Here are a few points:

* The only place `ADD_DOMAINS` are used is [here](https://github.com/mailserver2/mailserver/blob/master/rootfs/usr/local/bin/setup.sh#L58) it does not affect anything else
* I'm also running multiple domains and I did not have to do any changes
* Traefik 1 configuration did not require any special handling for multiple domains so chances are Traefik 2 does not require either
* Having said that, if you want to expose the same web site at multiple domains, you of course will have to tell traefik which these are. For my purposes I found a single URL for all mail domains sufficient. If you do not, you need to modify your configuration accordingly.

I hope this clears this out.

Reply:
Perfect! And sorry if I wasn't clear enough!